diff --git a/profiles/openberkeley/openberkeley_tests/behat/features/panopoly/spotlight_widget.feature b/profiles/openberkeley/openberkeley_tests/behat/features/panopoly/spotlight_widget.feature
index 34f7b20..7663d82 100644
--- a/profiles/openberkeley/openberkeley_tests/behat/features/panopoly/spotlight_widget.feature
+++ b/profiles/openberkeley/openberkeley_tests/behat/features/panopoly/spotlight_widget.feature
@@ -12,7 +12,7 @@ Feature: Add spotlight widget
       And I click "Add spotlight" in the "CTools modal" region
     Then I should see "Configure new Add spotlight"
 
-  @api @javascript @panopoly_widgets
+  @api @javascript @panopoly_widgets @chrome
   Scenario: Add a spotlight
     When I fill in the following:
       | field_basic_spotlight_items[und][0][title] | Testing item title  |
@@ -29,7 +29,7 @@ Feature: Add spotlight widget
       # Per an old bug described in issue #2075903
       And I should not see "Spotlight"
 
-  @api @javascript @panopoly_widgets
+  @api @javascript @panopoly_widgets @chrome
   Scenario: Image is required per issue #2075903
     When I fill in the following:
       | field_basic_spotlight_items[und][0][title] | Testing item title  |
